docs/pages/guides/customizing-webpack.mdx

```diff
@@ -81,9 +80,9 @@ The reason it automatically includes the polyfill is because `react-native-web`
 You can also use `npx expo customize` to generate the static project files: **index.html**, **serve.json**, etc.
```

Comment:
```suggestion
You can also use `npx expo customize` to generate the static project files: **index.html**, **serve.json**, and so on.
```

Unrelated to the changes in the PR, we can avoid [Latin abbreviations as per our writing style guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#abbreviations).